cmd/captplanet/setup.go

```diff
@@ -54,10 +54,9 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 		return err
 	}
 
-	valid, err := fpath.IsValidSetupDir(setupDir)
+	valid, _ := fpath.IsValidSetupDir(setupDir)
```

Comment:
Why are you removing `err`? Its used in `} else if setupCfg.Overwrite && err == nil {`